objects/client.c

```diff
@@ -2854,135 +3371,135 @@ client_class_setup(lua_State *L)
      * @signal mouse::move
      */
     signal_add(&client_class.signals, "mouse::move");
-    /**
+    /*
```

Comment:
I only looked at the "total diff", not the individual commits and have to say: Thanks for working on this.

The only small remark that I have is on this part here. This comment exists just for ldoc. If you "un-ldoc" it, it can just be removed instead.